src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/taskmanagementservicetests/NewInitiateTaskTest.java

```diff
@@ -149,6 +149,7 @@ void given_initiateTask_then_task_is_saved() {
         OffsetDateTime dueDate = OffsetDateTime.now();
         String formattedDueDate = CAMUNDA_DATA_TIME_FORMATTER.format(dueDate);
         initiateTaskRequest.getTaskAttributes().put(DUE_DATE.value(), formattedDueDate);
+        initiateTaskRequest.getTaskAttributes().put("taskId", taskId);
```

Comment:
I think the Task Monitor has to make sure that the taskId value is populated correctly.  Is this done already?